logging/log.go

```diff
@@ -391,10 +391,7 @@ func (l logger) GetTelemetrySession() string {
 }
 
 func (l logger) GetTelemetryHostName() string {
-	if !l.GetTelemetryEnabled() {
-		return ""
-	}
-	return l.loggerState.telemetry.telemetryConfig.getHostName()
+	return l.loggerState.telemetry.telemetryConfig.GUID
```

Comment:
I think we still want `getHostName()`, the optional name field is also static so this would only give us better information if available and still be static.